### ISSUE TITLE
Wrong visual about PointStamped radius

### ISSUE THREAD

**Issue by Julien-Livet** [ros-visualization/rviz#1676]
I observed a vocabulary mistake about PointStamped display : the field "Radius" currently matches to the diameter (with TF display).
I think it is more simple to rename the field than correct the code (div by 2), probably [https://github.com/ros-visualization/rviz/blob/5531206e77d8092dbdb6a50a5b1a3c23fe5e181e/src/rviz/default_plugin/point_display.cpp](https://github.com/ros-visualization/rviz/blob/5531206e77d8092dbdb6a50a5b1a3c23fe5e181e/src/rviz/default_plugin/point_display.cpp) or here [https://github.com/ros-visualization/rviz/blob/5531206e77d8092dbdb6a50a5b1a3c23fe5e181e/src/rviz/default_plugin/point_visual.cpp](https://github.com/ros-visualization/rviz/blob/5531206e77d8092dbdb6a50a5b1a3c23fe5e181e/src/rviz/default_plugin/point_visual.cpp).

![Capture d’écran_2021-11-02_11-38-21](https://user-images.githubusercontent.com/76999467/139822844-788931d2-fd68-40cf-a557-1b2ed55a6eb3.png)
### Your environment
* OS Version: e.g. Xubuntu 20.04
* ROS Distro: [Noetic]
* RViz, Qt, OGRE, OpenGl version as printed by rviz:
```[ INFO] [1635843067.996186502]: rviz version 1.14.10
[ INFO] [1635843067.996297233]: compiled against Qt version 5.12.8
[ INFO] [1635843067.996307080]: compiled against OGRE version 1.9.0 (Ghadamon)
[ INFO] [1635843068.006034471]: Forcing OpenGl version 0.
[ INFO] [1635843068.251274587]: Stereo is NOT SUPPORTED
[ INFO] [1635843068.251392943]: OpenGL device: llvmpipe (LLVM 12.0.0, 128 bits)
[ INFO] [1635843068.251541995]: OpenGl version: 3,1 (GLSL 1,4).```

**Comment by rhaschke** [comment]
Well spotted!

**Comment by rhaschke** [comment]
Fixed via #1678.
